hpat/hiframes/pd_series_ext.py

```diff
@@ -981,18 +981,22 @@ def generic_expand_cumulative_series(self, args, kws):
     return signature(return_type, recvr=self.this)
 
 
-# replacing cumsum/cumprod since arraydecl.py definition uses types.Array
-for fname in ["cumsum", "cumprod"]:
+# replacing cumprod since arraydecl.py definition uses types.Array
+for fname in ["cumprod"]:
     install_array_method(fname, generic_expand_cumulative_series)
 
 # TODO: add itemsize, strides, etc. when removed from Pandas
 _not_series_array_attrs = ['flat', 'ctypes', 'itemset', 'reshape', 'sort', 'flatten']
 
+# Array attributes which overlap Series attributes
+_excluded_array_attrs = ['cumsum']
```

Comment:
did you try just add `cumprod` into `_not_series_array_attrs ` array?

Reply:
Yes I did. The method isn't skipped because condition `attr not in _not_series_array_attrs` is `False` due to `attr = 'resolve_cumsum'` but not just `'cumsum'`. It looks like an issue, because now this condition is always `True`. Perhaps `attr.replace('resolve_', '')` might be useful in this case.